index.html

```diff
@@ -51,48 +55,64 @@
                   JavaScript can be extended for a variety of purposes by
                   supplementing it with additional objects; for example:
                 </p>
+
                 <li>
+
```

Comment:
- [ ] Kindly consider the `ul `tag for the list items `li  `you have included, which should be parenting the li items. This would help you improve you layout